lib/bios/menus.robot

```diff
@@ -226,9 +226,13 @@ Get Index Of Matching Option In Menu
     [Documentation]    This keyword returns the index of element that matches
     ...    one in given menu
     [Arguments]    ${menu_construction}    ${option}    ${ignore_not_found_error}=${FALSE}
+    Set Suite Variable    ${MATCHES}    ${FALSE}
```

Comment:
No need for suite variable, please make it local (unless removed, see next comment).

Reply:
https://github.com/Dasharo/open-source-firmware-validation/pull/580/commits/b72630e8dadf673f020969f8803813b114d33c37#diff-6ad7c5fb97cb1099c7ecd42f15dde3caf06d38b9fe5244afb96394454b6d6bfaL229-R229